pandas/core/indexes/multi.py

```diff
@@ -1078,6 +1078,27 @@ def levshape(self) -> Shape:
 
     @property
     def codes(self) -> tuple:
+        """
+        Codes of the MultiIndex.
+
+        Codes are the integer labels for each level designating the position of the index value in a sorted list of all index values in that level.
```

Comment:
The list need not be sorted (though often is). I would also stay way from "label" here - I think that's used to designate a name for something that isn't necessarily based on integer position.

What do you think of something like:

> Codes are the position of the index value in the list of level values for each level.